command/genesis/genesis.go

```diff
@@ -18,78 +17,70 @@ import (
 	"github.com/mitchellh/cli"
 )
 
-const (
-	genesisFileName       = "./genesis.json"
-	defaultChainName      = "example"
-	defaultChainID        = 100
-	defaultPremineBalance = "0x3635C9ADC5DEA00000" // 1000 ETH
-	defaultConsensus      = "pow"
-)
-
 // GenesisCommand is the command to show the version of the agent
 type GenesisCommand struct {
 	UI cli.Ui
-	Meta
+	helper.Meta
 }
 
 // DefineFlags defines the command flags
 func (c *GenesisCommand) DefineFlags() {
-	if c.flagMap == nil {
+	if c.FlagMap == nil {
 		// Flag map not initialized
-		c.flagMap = make(map[string]helper.FlagDescriptor)
+		c.FlagMap = make(map[string]helper.FlagDescriptor)
 	}
 
-	if len(c.flagMap) > 0 {
+	if len(c.FlagMap) > 0 {
 		// No need to redefine the flags again
 		return
 	}
 
-	c.flagMap["data-dir"] = helper.FlagDescriptor{
-		Description: fmt.Sprintf("Sets the directory for the Polygon SDK data. Default: %s", genesisFileName),
+	c.FlagMap["dir"] = helper.FlagDescriptor{
+		Description: fmt.Sprintf("Sets the directory for the Polygon SDK genesis data. Default: ./%s", helper.GenesisFileName),
```

Comment:
You don't need `./` since `GenesisFileName` is already `./genesis.json`, so it outputs `Default: ././genesis.json`
```suggestion
		Description: fmt.Sprintf("Sets the directory for the Polygon SDK genesis data. Default: %s", helper.GenesisFileName),
```

Reply:
Fixed 👍